pkg/kubelet/kubelet.go

```diff
@@ -1516,6 +1548,29 @@ func (kl *Kubelet) canAdmitPod(pods []*api.Pod, pod *api.Pod) (bool, string, str
 	return true, "", ""
 }
 
+func (kl *Kubelet) canRunPod(pod *api.Pod) lifecycle.PodAdmitResult {
```

Comment:
Can I say that calling this new method as canRunPod, which is the same as what you are using below: pkg/kubelet/util.go::canRunPod(...) is very confusing to me?